bin/check-haproxy.rb

```diff
@@ -68,6 +68,16 @@ class CheckHAProxy < Sensu::Plugin::Check::CLI
          default: 25,
          proc: proc(&:to_i),
          description: 'Critical Percent, default: 25'
+  option :min_warn_count,
+         short: '-M COUNT',
+         default: 0,
+         proc: proc(&:to_i),
+         description: 'Minimum Server Warn Count, default: 0'
+  option :min_crit_count,
+         short: '-X COUNT',
+         default: 0,
+         proc: proc(&:to_i),
+         description: "Minimum Server Critical Count, default: 0"
```

Comment:
bin/check-haproxy.rb:80:23: C: Prefer single-quoted strings when you don't need string interpolation or special symbols.